airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -2820,6 +2823,229 @@ def _get_query_result(self) -> Dict:
         return query_results
 
 
+class BigQueryHookAsync(GoogleBaseHookAsync):
+    """Big query async hook inherits from GoogleBaseHookAsync class and connects to the google Big query"""
```

Comment:
Rewrite docs - for example the fact that this class inherits from googleBasehookAsync is repeating information from line above, "google Big query" - bad case